### ISSUE TITLE
[4.0.0-3] `useHotkeys` with meta results in strange behaviour 

### ISSUE THREAD

**Issue by lunacrafts** [JohannesKlauss/react-hotkeys-hook#792]
```
  useHotkeys("meta+k", () => {
    setOpen((state) => !state);
  });
```

This code works correctly only until first execution. After first execution, the callback is fired after meta key is pressed (not combination of meta+k).

It seems to be a case only for meta. `ctrl+k` works fine.

**Comment by JohannesKlauss** [comment]
Thank you for reporting the issue. I'll have a look into it :)

**Comment by lunacrafts** [comment]
Also I'm not sure If It's bug or docs are outdated but 'esc' hotkey don't work, only 'escape' do.
Regarding this docs both should worl.
https://github.com/JohannesKlauss/react-hotkeys-hook/blob/v4.0.0-3/documentation/docs/documentation/useHotkeys/basic-usage.mdx

**Comment by JohannesKlauss** [comment]
The documentation for version 4 isn't updated yet, but both should work, yes

**Comment by lunacrafts** [comment]
Something is broken not only with meta key.

I use following code
  useHotkeys(
    "escape",
    (e) => {},
    {
      enableOnFormTags: ["INPUT"],
      enabled: open,
    }
  );
  
  and after first callback execution, callback is being executed on every key down

**Comment by JohannesKlauss** [comment]
Both issues should now be resolved with `4.0.0-4`

**Comment by lunacrafts** [comment]
Thank you  !! 🥳
